composer.json

```diff
@@ -50,7 +50,8 @@
         "mezzio/mezzio-helpers": "^5.3"
     },
     "require-dev": {
-        "composer/composer": "^1.5",
+        "composer/composer": "^1.5 || ^2.0@dev",
+        "composer/semver": "^1.5.1 || ^2.0@dev",
```

Comment:
I believe it is no longer needed, as we do have stable semver 2.0

Reply:
I removed the composer/semver